src/components/Actions.js

```diff
@@ -0,0 +1,12 @@
+function Actions(props){
```

Comment:
Since you are not using information coming from `props`, you don't need to pass the 'props' argument to the `Actions` function.